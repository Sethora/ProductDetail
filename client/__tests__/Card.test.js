import React from 'react';
import { shallow } from 'enzyme';
import Card from '../src/components/carousel/card';

describe('Card', () => {
  test('render the card', () => {
    const wrapper = shallow(<Card
      reference={() => { }}
      cards={[]}
      scroll={() => { }}
      click={() => { }}
      height={70}
      width={70}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});