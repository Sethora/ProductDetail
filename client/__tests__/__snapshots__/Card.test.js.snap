// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render the card 1`] = `
<styled.div
  height={70}
  onScroll={[Function]}
  width={70}
/>
`;
